README.md

```diff
@@ -1 +1,70 @@
-# bookStore
```

Comment:
- [ ] Kindly format your README using [the Microverse template](https://github.com/microverseinc/readme-template)